src/features/click-to-load.js

```diff
@@ -910,6 +911,12 @@ function unblockClickToLoadContent (message) {
  *   The parameters passed to acceptFunction when it is called.
  */
 function handleUnblockConfirmation (platformName, entity, acceptFunction, ...acceptFunctionParams) {
+    // Even if the user cancels the login attempt, consider Facebook Click to
+    // Load to have been active on the page if the user reports the page as broken.
+    if (entity === 'Facebook, Inc.') {
+        ctl.messaging.notify('updateFacebookCTLBreakageFlags', { ctlFacebookLogin: true })
+    }
```

Comment:
`handleUnblockConfirmation` might not only be used for login in the future. Might be best to set the flag either only in the `runLogin` or prior this message being called.

Reply:
Ah right, that's a good point. Thing is I still want to set the flag if the user clicks a login button, but then cancels the modal. I don't think `runLogin` is called until the user accepts the modal. Is there a better place to put this? I wasn't sure where was best to be honest.

Reply:
Yes, you're right, `runLogin` might be too late then.
I'd suggest adding it right before this function is called then. It would be [here](https://github.com/duckduckgo/content-scope-scripts/blob/main/src/features/click-to-load.js#L506) and [here](https://github.com/duckduckgo/content-scope-scripts/blob/main/src/features/click-to-load.js#L1848).

Reply:
Got it, thanks. Done.